### PR TITLE
Preserve crlf line endings of code.

### PR DESCRIPTION
**Pull request by jdalton**
I accidentally the PR. Related to #118.

### CODE REVIEW

File: lib/import-export-visitor.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const MagicString = require("./magic-string.js");
 const utils = require("./utils.js");
 const Visitor = require("./visitor.js");
 
+const carrageReturnCode = "\r".charCodeAt(0);
```
Comment (benjamn) [inline lib/import-export-visitor.js]: `carriageReturnCode`?

Comment (jdalton) [inline lib/import-export-visitor.js]: The `\r` is a [carriage return](https://en.wikipedia.org/wiki/Carriage_return).

Comment (jdalton) [inline lib/import-export-visitor.js]: Ah I typoed.

**Comment by jdalton** [comment]
Win!

**Comment by jdalton** [comment]
Cool to merge?

**Pull request merged**